pkg/cmd/boot/boot.go

```diff
@@ -98,6 +100,7 @@ func NewCmdBoot(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&options.EndStep, "end-step", "e", "", "the step in the pipeline to end at")
 	cmd.Flags().StringVarP(&options.HelmLogLevel, "helm-log", "v", "", "sets the helm logging level from 0 to 9. Passed into the helm CLI via the '-v' argument. Useful to diagnose helm related issues")
 	cmd.Flags().StringVarP(&options.RequirementsFile, "requirements", "r", "", "requirements file which will overwrite the default requirements file")
+	cmd.Flags().StringVarP(&options.EnvironmentRepo, "environment-repository-url", "n", "", "the  dev environment url repository to boot from")
```

Comment:
'-n' as shortcut? That seems a bit odd.

Also "the  dev environment url repository to boot from" -> "the URL of the dev environment repository to boot from"

I am wondering though how this is supposed to work. So `--environment-repository-url` allows you to specify a repo URL and it clones and boot from there, ignoring `--git-url` and `--git-ref` which are about specifying a env repo to fork from. Are these options not exclusive to each other? I seem to be missing how this is handled in code. 

It would be nice to have an issue as well which describes the context of this change and what it tries to achieve.

Reply:
I misunderstood what @garethjevans wanted for this PR, so I'm reworking it now. The idea is to boot from a previously created dev repository and not from boot-config. This will feed into being able to repair, re-install or upgrade an existing cluster.

There is an issue but not on jx, I'll add one with some context.

I agree "-n" is a bit odd, unfortunately the obvious ones ("e", "d", "r") . I'll just remove the shorthand.

Marking as WIP for now